srcpkgs/lsvpd/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'lsvpd'
 pkgname=lsvpd
 version=1.7.9
-revision=1
+revision=2
```

Comment:
please update this to `1.7.10` instead of bumping rev

to bypass the `xlint` problem, move the comment next to `license` on its own line (before `license`)

Reply:
lsvpd updated to 1.7.10 and successfully built locally. The license comment was moved up a line, but xlint still complains about vlicense.